src/pages/404.js

```diff
@@ -1,26 +1,47 @@
 import React from 'react';
+import { ERKENNINGEN_ADMIN_URL } from '@erkenningen/config';
 import FaqSidebar from '../components/FaqSidebar';
 import Layout from '../components/layout';
 
-const NotFoundPage = () => (
-  <Layout>
-    <div className="container">
-      <div className="row mb-5">
-        <div className="col-md-1 navbar-be__breadcrumbs-spacer-orange" />
-        <div className="col-md-2 navbar-be__breadcrumbs-spacer-green" />
-        <div className="col-md-9 navbar-be__breadcrumbs-container">Niet gevonden</div>
-      </div>
-      <div className="row">
-        <div className="col-md-3 pr-5">
-          <FaqSidebar />
+const NotFoundPage = () => {
+  // Determine if original request was a link to the old site
+  // If so, then show warning and add option to redirect to the administration site
+  const isOldLink = () => {
+    return typeof window !== 'undefined' && window.location.pathname.includes('/Default.aspx');
+  };
+  const getOldPath = () => {
+    return typeof window !== 'undefined' && window.location.pathname + window.location.search;
+  };
+
+  return (
+    <Layout>
+      <div className="container">
+        <div className="row mb-5">
+          <div className="col-md-1 navbar-be__breadcrumbs-spacer-orange" />
+          <div className="col-md-2 navbar-be__breadcrumbs-spacer-green" />
+          <div className="col-md-9 navbar-be__breadcrumbs-container">Niet gevonden</div>
         </div>
-        <div className="col-md-9 pl-0">
-          <h1>Pagina niet gevonden</h1>
-          <p>Wat jammer, de gezocht pagina bestaat niet. Kies uit het menu een pagina.</p>
+        <div className="row">
+          <div className="col-md-3 pr-5">
+            <FaqSidebar />
+          </div>
+          <div className="col-md-9 pl-0">
+            <h1>Pagina niet gevonden</h1>
+            {isOldLink() ? (
+              <p>
```

Comment:
Looks good! Maybe put this in an alert-info class? And also mention that the user should update his favorites/bookmarks?

Reply:
Check, or maybe alert-warning which is also better looking because of the color scheme

Reply:
Sure!

Reply:
Took some time because the build failed when using ui components in the 404 page. Apparently it is statically compiled as 404.html and therefore limited using React components which rely on window internally.

Reply:
Weird? Ok

Reply:
Maybe can you update the headers (breadcrumb + Panel title) to 'Pagina verhuisd' instead of 'Pagina niet gevonden'?